biometricks/src/main/java/com/handstandsam/biometricks/internal/UiHelpers.kt

```diff
@@ -20,6 +20,8 @@ internal object UiHelpers {
         showLoading: (Boolean) -> Unit,
         showPrompt: suspend () -> T
     ): T {
+        ensureFocus(activity)
```

Comment:
If moving this in here, this method should probably be renamed since it doesn't just do api 28 stuff.

Reply:
Good call, updating.